### DISCUSSION TITLE
Dropzone component does not work in React 17.0.0

### DISCUSSION THREAD

**Discussion by huaming422** [react-dropzone/react-dropzone#1275] [Q&A] [answered]
Currently, I am making the React Drag and Dropzone"
But I cannot use this with React 17.0.0

Please help me

**Comment by amaziah-yahalom** [comment, marked as answer]
Ensure that you are using a version of React Drag and Dropzone that is compatible with React 17.0.0. Sometimes, newer versions of libraries add support for newer versions of React.

**Comment by jimbeeth** [comment]
If you're using popular libraries like react-dnd or react-dropzone, check their documentation or GitHub page for React 17 support.
